app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -146,13 +146,12 @@ def redirect_to_vendor_outage_if_phone_only
     end
 
     def capture_analytics_for_exception(telephony_error)
-      attributes = {
-        error: telephony_error.class.to_s,
-        message: telephony_error.message,
-        context: context,
-        country: parsed_phone.country,
-      }
-      analytics.track_event(Analytics::OTP_PHONE_VALIDATION_FAILED, attributes)
+      analytics.otp_phone_validation_failed(
+          error: telephony_error.class.to_s,
+          message: telephony_error.message,
+          context: context,
+          country: parsed_phone.country,
```

Comment:
let's fix the indentation here
```suggestion
        error: telephony_error.class.to_s,
        message: telephony_error.message,
        context: context,
        country: parsed_phone.country,
```

Reply:
Updated please check